docs/guides/08-debugging/application.md

```diff
@@ -40,19 +40,59 @@ error: aborting due to previous error
 For more information about this error, try `rustc --explain E0425`.
 ```
 
-## WebView JS Console
+## WebView Console
 
 Right-click in the WebView, and choose `Inspect Element`. This opens up a web-inspector similar to the Chrome or Firefox dev tools you are used to.
+You can also use the `Ctrl+Shift+i` shortcut on Linux and Windows, and `Command+Option+i` on macOS to open the inspector.
```

Comment:
`Cmd+Option+I` doesn't work on macOS, at least on my OS version

Reply:
Actually, I take it back, working for me on macOS 12.4, Tauri-api rc 6 and Tauri-cli rc 13

Reply:
Must be an OS version thing then, it's not working for me on macOS 10.15

Reply:
I think the wry implementation only uses a newer API (the devtools API is private so apple can change it a lot). please file an issue on the wry repo @probablykasper I think I can set up a VM and try to fix it

Reply:
Isn't the `Cmd+Opt+I` shortcut provided by the webview itself? `window.open_devtools()` works fine, it's just the shortcut that doesn't work

Reply:
it's not provided by the webview, we're doing it

Reply:
ah, nice